book/website/reproducible-research/rdm/rdm-elns.md

```diff
@@ -0,0 +1,234 @@
+  Or perhaps a foundation or other non-profit/academic organisation with robust funding.
+- What are the data export options and formats?
+
+Open solutions provide you with the assurance that if you do the appropriate preparatory work you should be able to access all of your data in it's native form.
+For example keeping copies of the ELN software that can be run offline in a VM or similar computational environment in the future.
+This is typically much easier for open applications and can be impossible for proprietary ones.
+Even if the tools are no longer maintained and in a state that can be used in production they can still be used to read the data and interact with it in (mostly) the same way.
+The data will also likely be stored in an open format from which it can relatively easily be extracted and ported to a new format.
+This may still be a lot of work but it is much more attainable than reverse engineering a proprietary format for which there is no documentation or worse - dealing with encrypted files.
+
+You can pay for third party hosting and administration of open source applications.
+This is a solution when you don't have the expertise or internal resources to administer a self-hosted instance of an open-source ELN solution. 
+In this case you get the best of both worlds, the benefits of professional support and the reassurance of an open solution.
+You should still take regular local backups of exports from your hosting provider from which you could restore your ELN system with different hosting.
+This means that you retain the option to change providers.
+Conventionl software as a service (SaaS) platforms are highly verically integrated, the software development, hosting, and administration are package deal that you must take or leave as is.
+Using open tools breaks this up affording you the flexibility to change any of these if the need arises and you have the resources to do so, it puts you in a better bargining position.
+
+(rr-rdm-elns-choosing-automation)=
+### Automation and integration
+
+ELNs with APIs have the potential for integration with instruments that would permit direct deposition of data from connected instruments into ELNs or other connected research data management infrastructure.
+These sorts of integrations are especially useful in workflows at core facilities such as bioimaging facilities who can do things like automate a substancial part of the image metadata deposition into image data management tools such as [OMERO](https://www.openmicroscopy.org/omero/).
+
+(rr-rdm-elns-standards)=
+## ELN standards and Project governance
+
+ELNs offer many advantages over paper notebooks but they come with a considerable amount of additional complexity and a number of additional risks that it is important to have a plan to mitigate when proceeding with their adoption.
+The ELN space is still quite young and rapildy changing.
+ELNs represent a key component of digital infrastructure for the future of research.
```

Comment:
```suggestion
```

Reply:
Did this in: [327fba9](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/327fba994f8b0e9f0d44d30deeaf90b7858478eb)